site/content/docs/getting-started/setup-creds.md

```diff
@@ -1,40 +1,54 @@
-# Setting up credentials for the deployment tool<a name="deployment-tool-setup-creds"></a>
+# Setting up credentials
 
-****  
+AWS.Deploy.Tools, internally uses a variety of different tools and services to host your .NET application on AWS. To run the AWS Deploy Tool, you must configure a credential profile that provides access to the AWS account you wish to deploy to. Your credentials must have permissions for certain services, depending on the tasks that you're trying to perform.
 
-The information shown below is about how to set up credentials for the deployment tool. If you're looking for information about setting up credentials for your .NET project, see [Configure AWS credentials](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-config-creds.html) in the [AWS SDK for .NET Developer Guide](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-config-creds.html) instead.
+### Recommended policies
 
-To run the deployment tool against your AWS account, you must have a credentials profile in your shared AWS config and credentials files. The profile must be set up with at least an access key ID and a secret access key for an AWS Identity and Access Management (IAM) user. There are various ways to do this. For information see the following references:
+The following are some examples of the typical permissions that are required.
 
-- [Create users and roles](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-users-roles.html) and [Using the shared AWS credentials file](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/creds-file.html) in the AWS SDK for .NET Developer Guide.
-- [Providing AWS credentials](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/credentials.html) in the [AWS Toolkit for Visual Studio User Guide](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/welcome.html).
-- [Configuration and credential file settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) in the [AWS Command Line Interface User Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html).
+  > *Note: Additional permissions might be required, depending on the type of application you're deploying and the services it uses.*
 
-The credentials that you use to run the deployment tool must have permissions for certain services, depending on the tasks that you're trying to perform. The following are some examples of the typical permissions that are required to run the tool. Additional permissions might be required, depending on the type of application you're deploying and the services it uses.
+|Command| Task | Recommended AWS Managed Policies |
+| --- | --- |--- |
+|deploy | Deploying to Amazon ECS | AWSCloudFormationFullAccess, AmazonECS_FullAccess, AmazonEC2ContainerRegistryFullAccess, IAMFullAccess |

```

Comment:
None of these combinations will get you past the CDK bootstrap based on my testing last week.

I'm okay for now (especially given _"Additional permissions might be required"_ above), but we should probably take a follow-up task to document the bootstrap better.